.github/workflows/tests.yml

```diff
@@ -114,3 +114,58 @@ jobs:
           docker run --rm -e COMPOSER_ROOT_VERSION -v $(pwd):/app -v $(which composer):/usr/local/bin/composer -v /usr/local/bin/vulcain:/usr/local/bin/vulcain -w /app php:7.4-alpine ./phpunit src/Symfony/Component/HttpClient/Tests/CurlHttpClientTest.php --filter testHttp2Push
           sudo rm -rf .phpunit
           [ -d .phpunit.bak ] && mv .phpunit.bak .phpunit
+
+  nightly:
+    name: PHPUnit on PHP nightly
+    runs-on: Ubuntu-20.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          coverage: "none"
+          ini-values: "memory_limit=-1"
+          php-version: "8.1"
+
+      - name: Configure composer
+        run: |
+          COMPOSER_HOME="$(composer config home)"
+          composer self-update
+          ([ -d "$COMPOSER_HOME" ] || mkdir "$COMPOSER_HOME") && cp .github/composer-config.json "$COMPOSER_HOME/config.json"
+          echo "COMPOSER_ROOT_VERSION=$(grep -m1 SYMFONY_VERSION .travis.yml | grep -o '[0-9.x]*').x-dev" >> $GITHUB_ENV
+
+      - name: Install dependencies
+        run: |
+          echo "::group::fake PHP version"
+          composer config platform.php 8.0.99
+          echo "::group::composer update"
+          composer update --no-progress --ansi
+          echo "::endgroup::"
+          echo "::group::install phpunit"
+          ./phpunit install
+          echo "::endgroup::"
+
+      - name: Run tests
+        run: |
+          _run_tests() {
+            ok=0
+            echo "::group::$1"
+
+            # Run the tests
+            ./phpunit --colors=always --exclude-group tty,benchmark,intl-dat ./$1 2>&1 || ok=1
```

Comment:
`intl-data` (that might reclaim the extra 10 minutes?)

Reply:
I feel so stupid right now. 🙈